labs/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for Labs project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "labs.settings")
+
+application = get_asgi_application()
```

Comment:
how does this work when we have WSGI on the root of the project? Do we still have access to the fastapi endpoints?